stable/aws-for-fluent-bit/values.yaml

```diff
@@ -141,6 +164,74 @@ elasticsearch:
   # extraOutputs: |
   #   Index = my-index
 
+s3:
+  enabled: true
+  match: "*"
+  bucket:
+  region: "us-east-1"
+  jsonDateKey: "date"
+  jsonDateFormat: "iso8601"
+  totalFileSize: "100M"
+  uploadChunkSize: 50M
+  uploadTimeout: "10m"
+  storeDir: "/tmp/fluent-bit/s3"
+  storeDirLimitSize: 0
+  s3KeyFormat: /fluent-bit-logs/$TAG/%Y/%m/%d/%H/%M/%S
+  s3KeyFormatTagDelimiters:
+  staticFilePath: false
+  usePutObject: false
+  roleArn:
+  endpoint:
+  stsEndpoint:
+  cannedAcl:
+  compression:
+  contentType:
+  sendContentMd5: false
+  autoRetryRequests: true
+  logKey:
+  preserveDataOrdering: true
+  storageClass:
+  retryLimit: 1
+  externalId:
+  # extraOutputs: |
+
+opensearch:
+  enabled: true
+  match: "*"
+  host:
+  port: "443"
+  tls: "on"
+  path: ""
+  bufferSize: "5m"
+  pipeline:
+  awsRegion: "us-east-1"
+  awsAuth: "On"
```

Comment:
we just release AWS_Service_Name parameter for serverless support, you might as well add it here: https://github.com/aws-samples/amazon-ecs-firelens-examples/blob/mainline/examples/fluent-bit/amazon-opensearch-serverless/task-definition.json#L37

First added in this release: https://github.com/aws/aws-for-fluent-bit/releases/tag/v2.30.0

If we go with stable 2.28.4 for this chart (we should I think) then we can still add this option if we just call out that user needs to modify version themselves in the readme

this way we save time in the future and don't need a separate PR to add this

Reply:
Done